lib/descriptor.js

```diff
@@ -24,3 +26,67 @@ const generateDescriptor = async (data, dpName = "scratchpad") => {
   return descriptor
 }
 module.exports.generateDescriptor = generateDescriptor
+
+const generateDescriptorForLocalFiles = function (dpName = "scratchpad") { 
+  return new Promise(function(resolve, reject) { 
+    fs.readdir('./', (err, files) => {
+      var descriptor = {
+          name: dpName,
+          resources: []
+      };
+      files.forEach((file) => {
+        if (path.extname(file) == ".csv") {
+          var name = path.basename(file, '.csv').replace(/ /g, "-").toLowerCase();
+          fs.readFile(file, (err, data) => {
+            if(err) throw reject(err)
+            parse(data, (error, values) => {
+              if(error) throw reject(error)
+              var headers = values.shift()
+                , schema = infer(headers, values) 
+                descriptor.resources.push(
+                  { 
+                  "path" : file,
+                  "name" : name,
+                  "format": "csv",
+                  "schema": schema
+                }
+                )
+            })
+            
+            resolve(descriptor);   
+          })
+        } else if (path.extname(file) == ".json") {
+          var name = path.basename(file, '.json').replace(/ /g, "-").toLowerCase()
+          descriptor.resources.push(
+            { 
+              "path" : file,
+              "name" : name,
+              "format": "json"
+            }
+          )
+        } else if (path.extname(file) == ".geojson") {
+          var name = path.basename(file, '.geojson').replace(/ /g, "-").toLowerCase()
+          descriptor.resources.push(
+            { 
+              "path" : file,
+              "name" : name,
+              "format": "geojson"
+            }
+          )       
+        } else if (path.extname(file) == ".topojson") {
```

Comment:
I think it will be useful to use JS switch statement here instead of so many else if statements. 

Also what should we do if there is a file with extname that is not provided above?